account_payment_cancel/README.rst

```diff
@@ -0,0 +1,20 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Cancel account payment
+=================================
+
+This module allow you to cancel and remove a payment from a payment order or
+to free an invoice and put it in a new payment order.
+
+** Features list :**
+
+
+Known issues / Roadmap
+======================
+Make tests
+
+Contributors
+------------
```

Comment:
Please update the README to the latest model: [https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)